test/absinthe/schema/type_system_directive_test.exs

```diff
@@ -179,8 +183,7 @@ defmodule Absinthe.Schema.TypeSystemDirectiveTest do
   end
 
   @macro_schema_sdl """
-  "Represents a schema"
-  schema {
+  schema @feature(name: ":schema") {
```

Comment:
Does this need `extend`?  or for SDL is it support to declare this without extend?

Reply:
Turns out that SDL supports it without extend.